packages/dialect-react-ui/components/Notifications/EmailForm.tsx

```diff
@@ -49,6 +46,10 @@ export function EmailForm() {
   const [isEmailEditing, setEmailEditing] = useState(!emailObj?.enabled);
   const [emailError, setEmailError] = useState<ParsedErrorData | null>(null);
 
+  const [verificationCodeError, setVerificationCodeError] = useState<Error | null>(null);
```

Comment:
MIght be a weird question, but why are we handling these errors separately. I believe they can't happen simultaneously. Why not store them in a normalized matter (convenient for the UI) in one useState? wdyt?

Reply:
Ah, it was done like that before. I think we should refactor, wdyt?

Reply:
Agreed! Additional q: why did you move the errors from ApiContext to the EmailForm itself at the first place? Since we would have SMS and telegram it make sense to keep it in one place. 

@AlekseiPetrovDev, wdyt?